packaging/windows/install_script.nsis

```diff
@@ -0,0 +1,237 @@
+Unicode true
```

Comment:
Is there any way to test this file without having to spin up a Windows VM and install the agent?

Reply:
You can test the creation to ensure it spits out a file that "looks" correct, ie it should be 30ish MB and a valid exe. But aside from that, I am not aware of anything.